source/extensions/omni.isaac.lab/omni/isaac/lab/sim/simulation_cfg.py

```diff
@@ -24,22 +24,16 @@ class PhysxCfg:
     documentation`_.
 
     PhysX 5 supports GPU-accelerated physics simulation. This is enabled by default, but can be disabled
-    through the flag `use_gpu`. Unlike CPU PhysX, the GPU simulation feature is not able to dynamically
-    grow all the buffers. Therefore, it is necessary to provide a reasonable estimate of the buffer sizes
-    for GPU features. If insufficient buffer sizes are provided, the simulation will fail with errors and
-    lead to adverse behaviors. The buffer sizes can be adjusted through the `gpu_*` parameters.
+    by setting the :attr:`~SimulationCfg.device` to `cpu` in :class:`SimulationCfg`. Unlike CPU PhysX, the GPU
+    simulation feature is not able to dynamically grow all the buffers. Therefore, it is necessary to provide
+    a reasonable estimate of the buffer sizes for GPU features. If insufficient buffer sizes are provided, the
+    simulation will fail with errors and lead to adverse behaviors. The buffer sizes can be adjusted through the
+    `gpu_*` parameters.
```

Comment:
The ticks were misplaced.

```suggestion
    by setting the :attr:`~SimulationCfg.device` to ``cpu`` in :class:`SimulationCfg`. Unlike CPU PhysX, the GPU
    simulation feature is unable to dynamically grow all the buffers. Therefore, it is necessary to provide
    a reasonable estimate of the buffer sizes for GPU features. If insufficient buffer sizes are provided, the
    simulation will fail with errors and lead to adverse behaviors. The buffer sizes can be adjusted through the
    ``gpu_*`` parameters.
```